addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureScanRule.java

```diff
@@ -765,21 +776,9 @@ private void findBackupFile(HttpMessage originalMessage) throws Exception {
                                                 != requestStatusCode
                                         && (!Arrays.equals(
                                                 disclosedData, nonexistparentmsgdata))))) {
-                    newAlert()
-                            .setConfidence(Alert.CONFIDENCE_MEDIUM)
-                            .setName(
-                                    Constant.messages.getString(
-                                            "ascanbeta.backupfiledisclosure.name"))
-                            .setAttack(candidateBackupFileURI.toString())
-                            .setOtherInfo(originalMessage.getRequestHeader().getURI().toString())
-                            .setSolution(
-                                    Constant.messages.getString(
-                                            "ascanbeta.backupfiledisclosure.soln"))
-                            .setEvidence(
-                                    Constant.messages.getString(
-                                            "ascanbeta.backupfiledisclosure.evidence",
-                                            originalURI,
-                                            candidateBackupFileURI))
+                    buildAlert(
```

Comment:
Is the `setName` call not needed?

Reply:
It's the same as `getName()` which is set by default.